cluster/options.go

```diff
@@ -4,8 +4,16 @@ import "crypto/tls"
 
 // Options is exported
 type Options struct {
-	TLSConfig       *tls.Config
+	TLS             *TLSConfig
 	OvercommitRatio float64
 	Discovery       string
 	Heartbeat       uint64
 }
+
+// TLSConfig is exported
+type TLSConfig struct {
```

Comment:
@vieux yes that would be cool if we could feed the etcd `http.Transport` with a `tls.Config` directly. As of now I think properties and methods doing this are private and the only external point of setup for TLS are path to cert files.